dcompiler.dd

```diff
@@ -864,6 +865,23 @@ make -f posix.mak DMD=~/$(DMDDIR)/$(OS)/bin/dmd
 )
 
 
+$(H2 $(LEGACY_LNAME2 tips_tricks, tips-tricks, Tips and Tricks))
+
+    $(P Get compiler info on command line (posix):)
+
+
+$(UNIX
```

Comment:
Please but the entire section and menu item in the `$(UNIX)` tag. Otherwise it will be empty and look weird for people on Windows.